CRM/Activity/BAO/Activity.php

```diff
@@ -2511,7 +2509,7 @@ public static function getContactActivitySelector(&$params) {
         }
 
         $activity['activity_date_time'] = CRM_Utils_Date::customFormat($values['activity_date_time']);
-        $activity['status_id'] = $activityStatus[$values['status_id']];
+        $activity['status_id'] = CRM_Core_Pseudoconstant::getLabel('CRM_Activity_BAO_Activity', 'status_id', $values['status_id']);
```

Comment:
As noted on the original this isn't the same and changes what displays for disabled. I found the earlier discussion, and it also points to an even earlier discussion: https://github.com/civicrm/civicrm-core/pull/14240#issuecomment-496791671

Reply:
ah - I'll close this out then - since the point was to peel off the easy bits